as/src/base/index.c

```diff
@@ -676,15 +692,14 @@ as_index_sprig_get_insert_vlock(as_index_sprig *isprig, uint8_t tree_id,
 
 		cf_arenax_handle t_h = isprig->sprig->root_h;
 		as_index *t = RESOLVE_H(t_h);
+		_mm_prefetch(t, _MM_HINT_NTA);
```

Comment:
commit message should also explain that the prefetch is happening as early as possible.

Reply:
done